src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from enum import Enum
+from typing import Dict, List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from serenity_types.refdata.options import OptionType
+
+
+class VolModel(Enum):
+    """
+    Currently supported volatility models.
+    """
+
+    SVI = "SVI"
+    """
+    Stochastic volatility (SVI) calibrated volatility model.
+    """
+
+    BLACK_SCHOLES = "BLACK_SCHOLES"
+    """
+    Classic Black-Scholes volatility model.
+    """
+
+
+class VolSurfaceType(Enum):
+    """
+    Representation of the volatility surface, e.g. raw inputs vs. fitted surface.
+    """
+
+    RAW = "RAW"
+    """
+    A volatility surface containing the raw inputs: the option prices, IV's, etc..
+    """
+
+    FITTED = "FITTED"
+    """
+    A calibrated and fitted volatility surface with a dense grid of IV's.
+    """
+
+
+class StrikeType(Enum):
+    """
+    Currently supported strike representations.
+    """
+
+    ABSOLUTE = "ABSOLUTE"
+    """
+    Absolute value of strike, e.g. the 20000 option.
+    """
+
+    LOG_MONEYNESS = "LOG_MONEYNESS"
+    """
+    Relative value of strike vs. current spot, with log transformation, e.g. 1.05
+    """
+
+
+class VolatilitySurface(BaseModel):
```

Comment:
Is this the expected return for getVolatilitySurfaces? or we need another VolSurfaceDefinition?
```

class VolSurfaceDefinition(BaseModel):
    """
    Basic metadata for volatility surfaces
    """

    vol_surface_id: UUID
    """
    The unique identifier for this surface
    """

    display_name: str
    """
    Human-readable descrition of this curve, e.g. OIS (RAW) or OIS (Interpolated, FLAT_FWD).
    """

    vol_model: VolModel
    """
    Volatility model used for this surface.
    """

    strike_type: StrikeType
    """
    Strike representation used for this surface, e.g. ABSOLUTE or LOG_MONEYNESS.
    """

    start_datetime: datetime
    """
    UTC datetime for the earliest version in the database.
    """

    end_datetime: datetime
    """
    UTC datetime for the latest version in the database.
    """

```

Reply:
You are right, I need to revisit the return values for the getVolatilitySurfaces and getYieldCurves calls. I will do this as part of the work today to incorporate the new types into the serenity.specifications notebook. Bear with me.